build.sbt

```diff
@@ -1,16 +1,20 @@
-val catsEffectVersion = "3.2.9"
 lazy val root = (project in file("."))
   .settings(publishOptions)
   .settings(
     name := "ciris-aws-ssm",
     organization := "com.ovoenergy",
     licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
-    scalaVersion := "3.1.0",
-    crossScalaVersions := Seq(scalaVersion.value, "2.13.1", "2.12.10"),
+    scalaVersion := "3.2.0",
```

Comment:
```suggestion
    scalaVersion := "3.1.3",
```

Reply:
Sorry, I am really not with it this week.  Thanks :)